app/lib/world/world.coffee

```diff
@@ -378,6 +378,12 @@ module.exports = class World
     return unless @goalManager
     @goalManager.submitWorldGenerationEvent(channel, event, @frames.length)
 
+  publishCameraEvent: (eventName, event) ->
+    return if not Backbone?.Mediator # headless mode don't have this
+    event ?= {}
+    channel = 'camera:' + eventName
```

Comment:
Should this be passed in as the first argument beneath?
Or maybe `eventName = 'camera:' + eventName` could also work.

Reply:
oh, hurry is a bad way. Thanks. Fixed